packages/frontend/src/pages/EpochKeyPage.jsx

```diff
@@ -25,63 +26,28 @@ export default observer(() => {
   const attestations = unirep.attestationsByEpochKey.get(epochKeyId)
 
   return (
-    <>
+    <body>
```

Comment:
@CJ-Rose do you have some docs on using the `body` tag in jsx? I'm concerned we'll end up with nested `body` elements during render. We render into an html template that looks like this (into the `id="root"` `div`)

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <link rel="icon" href="favicon.ico" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="theme-color" content="#000000" />
    <meta name="explorer" content="" />
    <link rel="apple-touch-icon" href="logo192.png" />
    <link
      rel="stylesheet"
      href="https://fonts.googleapis.com/css2?family=Azeret+Mono:wght@200..700&display=swap"
    />
    <title>explorer</title>
    <script>
      window.__DEV_CONFIG__ = '__DEV_CONFIG__'
    </script>
  </head>
  <body>
    <noscript>You need to enable JavaScript to run this app.</noscript>
    <div id="root"></div>
    <!--
      This HTML file is a template.
      If you open it directly in the browser, you will see an empty page.
      You can add webfonts, meta tags, or analytics to this file.
      The build step will place the bundled scripts into the <body> tag.
      To begin the development, run `npm start` or `yarn start`.
      To create a production bundle, use `npm run build` or `yarn build`.
    -->
  </body>
</html>
```

Reply:
yeah you're right. I had been fighting with the footer placement for too long when I stuck that `body` tag in there : /
Removed and created a new class for that div.